### PR TITLE
Update Copywriter and Change log dir

### PR DESCRIPTION
**Pull request by vndroid**
Before user need to authorize the plugin folder as a whole, you only need to authorize the log directory (such as: chown 777 ./logs) and update the plugin description information and inside copywriter.

### CODE REVIEW

File: logs/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+!.gitignore
```
Comment (oott123) [inline logs/.gitignore]: You should ignore log files here, e.g., `*_log.php` or just `*`

**Review by oott123 — CHANGES_REQUESTED** [review]

File: logs/.gitignore (hunk 1 above)
Comment (vndroid) [reply logs/.gitignore]: > You should ignore log files here, e.g., `*_log.php` or just `*`

`!.gitignore` can not work?

Comment (vndroid) [reply logs/.gitignore]: Okey

Comment (oott123) [reply logs/.gitignore]:
```
*
!.gitignore
```

is okay, however if you just use `!.gitignore` it won't work

**Review by oott123 — APPROVED** [review]

**Pull request merged**